system/jlib/jstatcodes.h

```diff
@@ -326,6 +326,10 @@ enum StatisticKind
     StNumPeakCacheObjects,              // Peak number of objects in a generic cache
     StNumCacheDuplicates,
     StNumCacheEvictions,
+    StNumDuplicateKeyCount,
+    StSizeOffsetBranches,
```

Comment:
Not a great name - but I can't think of anything that is much better.

SizeBeforeBranches.   SizeLeafAndBlobs    SizeNonBranches

none are worth replacing,